mkdocs.yml

```
@@ -1,7 +1,28 @@
 site_name: LORIS
 repo_url: https://github.com/aces/Loris
-pages:
-    - Home: index.md
-    - API: API/LorisRESTAPI_v0.0.3.md
-    - Wiki: wiki/index.md
+nav:
+    - Home: 'index.md'
+    - Wiki: 'wiki/index.md'
+    - Server Install and Configuration: 
```

This is the numbering I was asking if we could include in my prior comment --
it read wrong because it was missing a "not"
```suggestion
    - 0. Server Install and Configuration: 
```